Tasks/DockerV2/package.json

```diff
@@ -5,7 +5,7 @@
     "@types/q": "^1.5.0",
     "@types/uuid": "^8.3.0",
     "azure-pipelines-task-lib": "^3.0.6-preview.0",
-    "azure-pipelines-tasks-docker-common-v2": "2.0.6",
+    "azure-pipelines-tasks-docker-common-v2": "^2.198.1",
```

Comment:
Looks like this was previously pinned to some specific reason. Can you check the history why this was done such a way to be on safer side?

Reply:
Looks like the last time this file was updated was over a year ago, in which the docker-common-v2 file was updated from 2.0.4 to 2.0.6. I can revert this back to 2.0.6 if need be, but we may need to update this file in the future as 2.0.6 is slightly older than the current version of docker-common-v2 that is currently supported.